float32[] descriptor
geometry_msgs/Vector3 point3d
int32 u
int32 v